src/routes/tweets.route.ts

```diff
@@ -0,0 +1,84 @@
+// eslint-disable-next-line import/order
+import express, { Request, Response } from "express";
+
+import * as repo from "../repositories/tweet.repository";
+
+export const tweetsRouter = express.Router();
+const inputErrorMessage = "Input error";
+tweetsRouter.get("/getOne/:id", async (req: Request, res: Response) => {
+  try {
+    const tweet = await repo.getTweetById(req.params.id);
+    res.json(tweet);
+  } catch (error) {
+    if (error.message.includes(inputErrorMessage)) {
+      return res.status(400).json({ message: error.message });
+    } else {
+      return res.status(500).json({ message: error.message });
+    }
+  }
```

Comment:
```suggestion
    const tweet = await repo.getTweetById(req.params.id);
    res.json(tweet);
```
it is not necessary to use try ... catch. Our project has error handler middleware
`middlewares/errorHandler.middlewares.ts`
and it will catch the error and send. Additionaly it will log it.
You can check it

Reply:
cool thing, but how can i pass status code? i can only throw Error instanses(didn`t manage even to send a subclass of error)

Reply:
you can throw CustomError